src/main/java/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package strategy;
+
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void apply(Map<String, Integer> inventory, String fruit, int quantity) {
```

Comment:
Consider verifying that the `quantity` is not negative before applying it to the inventory, as returning a negative amount does not make sense logically and could potentially introduce errors in the inventory counts.